src/components/webviews/CozyWebView.js

```diff
@@ -136,6 +172,10 @@ export const CozyWebView = ({
     return () => BiometryEmitter.off('change', injectSettings)
   }, [injectSettings, webviewRef])
 
+  if (shouldShowCliskDevMode() && launcherContext.state === 'default') {
+    return <CliskDevView setLauncherContext={trySetLauncherContext} />
+  }
+
```

Comment:
By doing this is the CozyWebView, this means that both homescreen and cozyappscreen will display the CliskDevView when `shouldShowCliskDevMode`. So it will be displayed twice. Is that expected/problematic?

Reply:
PS: after some reflexion I indeed think that this may be problematic as we expect all our `CozyProxyWebView`, `CozyWebView`, `ReloadInterceptorWebView` (etc) to always return a WebView and to be interchangeable. So maybe we can keep this code in the HomeScreen where it was?

Reply:
Right, this is done now